README.md

```diff
@@ -1,13 +1,13 @@
-# [MyProxy](http://innout.life:3000/) &middot; [![CircleCI](https://circleci.com/gh/garageScript/myProxy.svg?style=svg)](https://circleci.com/gh/garageScript/myproxy)
+# [MyProxy](https://garagescript.github.io/myProxy/) &middot; [![CircleCI](https://circleci.com/gh/garageScript/myProxy.svg?style=svg)](https://circleci.com/gh/garageScript/myproxy)
 MyProxy is an application that proxies requests to other servers
 
 # Prerequisites
 
-- [x] To run **myProxy** first of all you'll need a server with **Node.js** and **Git** installed
+- [x] To run **myProxy** first of all you'll need a server with **Git** installed
 
-- [x] A valid Domain name `Eg: mydomain.com`
+- [x] A valid Domain name `Eg: mydomain.com` (Currently supported providers: GoDaddy)
 
-# How to install
+# How to use the app
```

Comment:
I think `install` is better. `use the app` sounds more like the section where you create a subdomain, git clone, edit, git push, etc.

Reply:
or... `install and use`

Or maybe break down the instruction into subsections? 
```
# Installation and usage

## Installation and Setup
...
...

## Using the app
After your domain is setup, you will be able to generate as many ....
1. blah blah...
```